src/bpfmap.h

```diff
@@ -14,6 +17,23 @@ namespace libbpf {
 
 namespace bpftrace {
 
+static const std::unordered_map<std::string, libbpf::bpf_map_type>
```

Comment:
Nit: Unless these are used in the header, I would place these definitions inside the implementation file.. If they are used in multiple compilation units, I would place the declaration here but the actual definition (non-static) in a reasonable home.